example/variables.tf

```diff
@@ -113,6 +113,30 @@ instances are launched.
 EOF
 }
 
+variable "router_name" {
+  type = "string"
+
+  description = <<EOF
+The name of the Google Compute Engine router.
+EOF
+}
+
+variable "address_name" {
+  type = "string"
+
+  description = <<EOF
+The name of the Google Compute Engine static address.
+EOF
+}
+
+variable "nat_name" {
+  type = "string"
+
+  description = <<EOF
+The name of the Google Compute Engine NAT.
+EOF
```

Comment:
Does this need to be a variable? Can you provide an example (in this description) of when you might want to override/manually specify it?

Reply:
This variable is now used to explicitly enable/disable the Cloud NAT as explained in the description.

It was to set the NAT resource's name which is now fixed based on the name of the cluster. The router resource is named in the same way.